setup.cfg

```diff
@@ -1,14 +1,22 @@
+[aliases]
+test=pytest
+
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503
+ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503, E203, F401, F403, F405
```

Comment:
Why are you ignoring F401, F403 and F405? Those are for unused modules being imported and for use of `from foo import *`, which should only be in `__init__.py` files (which are excluded at line 10).

I really hope you can re-enable these -- both here and in line 19. They are useful!

I tried it in my local copy and saw just one flake8 error -- a real one.

Reply:
I copied this from another project and indeed removing those reveals a real error. Thanks! I'll review my other projects as well.

Reply:
One more very minor suggestion: please consider moving E203 right after E133. With the ignore items sorted it is easier to see which things you are ignoring. Your new list (with E203 but not the F items) matches what salobj is using.